src/components/ManageDataModal.js

```diff
@@ -76,6 +92,29 @@ export default function ManageDataModal({ show, onClose }) {
                             )}
                         </div>
                     </Tab>
+                    <Tab eventKey="runelite" title="Runelite Import">
+                        <div className="text-center mt-3">
+                            <Alert variant="warning" className="small">
+                                <b>Note:</b> Importing from Runelite will only update tasks, relics, and areas.
+                            </Alert>
```

Comment:
Wonder if we should also add a small blurb here telling people how to install the plugin in case they don't know about external plugins

Reply:
Great point!! I'll throw something in.

Reply:
Updated with a link to the Plugin Hub wiki article.